app/templates/ErrorPage.twig

```diff
@@ -0,0 +1,20 @@
+<section class="container clearfix">
```

Comment:
We have a similar file (that goes by the same name) in the content management wiki. Did you introduce this one to not display the "try again later" notice?